### PR TITLE
Create data flash vue component

### PR DESCRIPTION
**Pull request by lipskij**
Add `dataflash` vue component
Will create a separate PR to integrate into configurator.

Screenshot 2023-07-10 at 18 11 21

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/components/data-flash/DataFlash.vue
Comment (haslinghuis) [inline src/components/data-flash/DataFlash.vue:4]: `suport` is misspelled in this PR. Please also update other occurrences.
```suggestion
      v-if="!supportDataflash"
```

**Review by haslinghuis — APPROVED** [review]

File: src/components/data-flash/DataFlash.vue
Comment (haslinghuis) [inline src/components/data-flash/DataFlash.vue:41]:
```suggestion
            if (this.fcUsedSize >= this.fcTotalSize) {
```

**Review by haslinghuis — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
